src/main/resources/db/migration/V3_1__add_expired_varsel_published_at_to_aktivitetskrav_varsel.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE AKTIVITETSKRAV_VARSEL
+    ADD COLUMN expired_varsel_published_at TIMESTAMPTZ DEFAULT NULL;
+
+ALTER TABLE AKTIVITETSKRAV_VARSEL
+    ADD COLUMN svarfrist DATE NOT NULL
+        DEFAULT (created_at + INTERVAL '3 weeks');
```

Comment:
Sorry, jeg har forsøkt å lese litt i doc'en og tror ikke Postgres tillater dette. 

Så jeg tror scriptet må utvides: 
1. Opprette som over med uten "NOT NULL"
2. Kjøre en update som setter svarfrist til created_at + INTERVAL '3 weeks'
3. Kjøre en ALTER TABLE ALTER COLUMN svarfrist SET NOT NULL;